### PR TITLE
Work on the LoadFile unqualifier section

### PR DESCRIPTION
**Pull request by sammoorhouse-MS**

### CODE REVIEW

**Comment by kthielen** [comment]
It looks like the macOS build failure was due to a timeout on the host prior to building hobbes (looks like it got pre-empted building LLVM).

Anyway, obviously that build failure is unrelated to anything you're changing here ...

File: doc/en/logging/hi.rst [hunk 1]
```diff
@@ -55,15 +55,33 @@ Therefore the annotation becomes, at compile time, a *type-safe* indication of t
...
 
+We can query data inside a logfile using the comprehensions syntax we saw :ref:`previously<comprehensions>`.
+
+Remember that our logger contains the following line:
+
+::
+
+  HSTORE(SimpleLogger, FirstEvent, "First", 0, 1, 2);
+
+Hobbes exposes this persisted element as a tuple, and so we can unpack it using the numbered indexing syntax for Hobbes tuples:
```
Comment (kthielen) [inline doc/en/logging/hi.rst]: Nice connection.

File: doc/en/logging/hi.rst [hunk 2]
```diff
@@ -71,13 +71,12 @@ Hobbes exposes this persisted element as a tuple, and so we can unpack it using
   ["12", "12", "12", "12", "12", "12", 
   ...
 
-  > [ messages.SecondEvent[0:3]]
-  [First 0 1 2 
+And although they type of ``messages.SecondEvent`` isn't quite an array, you can use the "slice" notation to work with a subset of logged messages:
```
Comment (kthielen) [inline doc/en/logging/hi.rst]: s/they/the

File: doc/en/logging/HLOG.rst [hunk 3]
```diff
@@ -39,7 +39,11 @@ The third parameter to ``DEFINE_STORAGE_GROUP`` is the behaviour to exhibit if w
...
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The final parameter to ``DEFINE_STORAGE_GROUP`` allows us to specify the commit behaviour. ``hobbes::storage::ManualCommit`` means that all log statements up to the ``commit`` call will be grouped together in a transaction, which we can inspect later. Alternatively, specifying ``hobbes::storage::AutoCommit`` gives us uncorrelated log messages on the consumer side, but we don't need to call ``commit`` ourselves.
+The final parameter to ``DEFINE_STORAGE_GROUP`` allows us to specify the commit behaviour. ``hobbes::storage::ManualCommit`` means that all log statements up to the ``commit`` call will be grouped together in a transaction, which we can inspect later.
+
+Alternatively, specifying ``hobbes::storage::AutoCommit`` gives us uncorrelated log messages on the consumer side, but we don't need to call ``commit`` ourselves.
+
+Transactions which are manually committed will have a timestamp logged alongside them which can be used in out-of-band analysis and reporting, whereas autocommitted transactions have no timestamp. You can always call ``commit`` on an autocommit LogGroup, which will immediately persist the current transaction.
```
Comment (kthielen) [inline doc/en/logging/HLOG.rst]: Maybe it's not worth mentioning but if people do want timestamps then they can log them.  :)

Comment (sammoorhouse-MS) [reply doc/en/logging/HLOG.rst]: I managed to convert a std::chrono::duration to epoch time which I could persist; is there a  better way?

**Pull request merged**